cmd/init.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2017 by the contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var initCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Initialize kubernetes-aws-authenticator by generating the key used by the server, a kubeconfig used by clients, the certificate used between both",
```

Comment:
Nit: maybe something like this?
`Pre-generate certificate, private key, and kubeconfig files for the server.`

I think "kubeconfig used by clients" is confusing here because the generated kubeconfig is only used for the apiserver webhook configuration (so it knows where to send authentication webhook requests). Clients like kubectl need a different kubeconfig that depends on the apiserver's CA.